docs/code-examples/arrow3d_simple.py

```diff
@@ -12,4 +12,4 @@
 vectors = np.column_stack([np.sin(angles) * lengths, np.zeros(100), np.cos(angles) * lengths])
 colors = [[1.0 - c, c, 0.5, 0.5] for c in angles / tau]
 
-rr2.log("arrows", rr2.Arrows3D(vectors, colors=colors))
+rr2.log("arrows", rr2.Arrows3D(vectors=vectors, colors=colors))
```

Comment:
Can we please add an origin here while we're at it? I think it makes the example a lot more useful to the user.